src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -445,14 +445,14 @@ protected virtual ActorStopped CreateActorStoppedEvent()
             return new ActorStopped(Self, Props.Type);
         }
 
-        private void Create(Exception failure)
+        private void Create(Exception? failure)
```

Comment:
The `Exception` was always optional on the `Create` method because during first initialization there won't be one.